backend/capellacollab/projects/capellamodels/routes.py

```diff
@@ -84,58 +83,66 @@ def create_new(
                 "technical": "Slug already used",
             },
         )
-    return ResponseModel.from_orm(model)
+    return model
```

Comment:
The return could be directly in the try:

Reply:
True, I moved it directly into the try block